src/playground.ts

```diff
@@ -19,7 +19,7 @@ import HLTV from './index'
 //     console.dir(data, { depth: null })
 // }})
 // HLTV.getMatchesStats({startDate: '2017-07-10', endDate: '2017-07-18'}).then(res => console.log(res.length))
-// HLTV.getMatch({id: 2312432}).then(res => console.dir(res, {depth: null})).catch(err => console.log(err))
+HLTV.getMatch({id: 2332985}).then(res => console.dir(res, {depth: null})).catch(err => console.log(err))
```

Comment:
Can you please discard the changes in this file?

Reply:
Done.